src/reader/schema.rs

```diff
@@ -278,10 +278,11 @@ pub fn create_schema(types: &[Type], root_column: usize) -> Result<Arc<TypeDescr
             );
 
             let td = Arc::new(TypeDescription::new(ARRAY.clone(), root_column));
-            let fields = &root.field_names;
-            for (idx, column) in sub_types.iter().enumerate() {
+
+            for (_, column) in sub_types.iter().enumerate() {
```

Comment:
If not using index then no need to enumerate?

Reply:
Good point!